app/src/main/java/org/gnucash/android/ui/transaction/TransactionsListFragment.java

```diff
@@ -100,8 +100,9 @@ public void onCreate(Bundle savedInstanceState) {
         Bundle args = getArguments();
         mAccountUID = args.getString(UxArgument.SELECTED_ACCOUNT_UID);
 
+        boolean isDoubleEntryDisabled = !GnuCashApplication.isDoubleEntryEnabled();
         mUseCompactView = PreferenceActivity.getActiveBookSharedPreferences()
-                .getBoolean(getActivity().getString(R.string.key_use_compact_list), !GnuCashApplication.isDoubleEntryEnabled());
+                .getBoolean(getString(R.string.key_use_compact_list), false) || isDoubleEntryDisabled;
```

Comment:
@pnemonic78 Isn't this the same as directly passing the fallback value to [getBoolean](https://developer.android.com/reference/android/content/SharedPreferences#getBoolean(java.lang.String,%20boolean))?

Reply:
no. fallback is when no value defined in prefs. 
always show compact when no double-entry.